chat-widget/src/components/livechatwidget/common/liveChatConfigUtils.ts

```diff
@@ -4,4 +4,11 @@ export const isPostChatSurveyEnabled = async (chatSDK: any) : Promise<boolean> =
     const postChatEnabled = chatConfig.LiveWSAndLiveChatEngJoin
         ?.msdyn_postconversationsurveyenable.toString().toLowerCase();
     return postChatEnabled === "true";
-};
\ No newline at end of file
+};
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export const isPersistentChatEnabled = async (chatSDK: any): Promise<boolean> => {
+    const chatConfig = await chatSDK.getLiveChatConfig();
+    const conversationMode = chatConfig.LiveWSAndLiveChatEngJoin?.msdyn_conversationmode.toString().toLowerCase();
+    return conversationMode === "192350001";
```

Comment:
set this in a constant 192350001 , so we know what is this number about

Reply:
Reused constants